app/helpers/comments_helper.rb

```diff
@@ -1,23 +1,40 @@
 module CommentsHelper
   def render_comment_tree(comment)
-    content_tag :li do
-      concat content_tag(:p, comment.text)
-      concat content_tag(:div, style: "display: flex; align-items: center;") {
-        safe_concat "Posted by "
-        safe_concat link_to(comment.user.name, comment.user)
-        safe_concat "  on #{comment.created_at.strftime("%B %d, %Y at %I:%M %p")}"
-        safe_concat " - Score: "
-        safe_concat content_tag(:span, comment.score, id: "score-Comment-#{comment.id}")
-
-        safe_concat button_to("Upvote", upvote_article_comment_path(comment.article, comment), method: :post, class: "btn btn-sm upvote-button", data: {votable_type: "Comment", votable_id: comment.id, article_id: comment.article.id})
-        safe_concat button_to("Downvote", downvote_article_comment_path(comment.article, comment), method: :post, class: "btn btn-sm downvote-button", data: {votable_type: "Comment", votable_id: comment.id, article_id: comment.article.id})
+    content_tag :div do
+      concat content_tag(:table, border: "0", cellpadding: "0", cellspacing: "0") {
```

Comment:
Because this is a Hacker News clone, it's fine to build this as a table (since Hacker News itself is a giant table) and have it do some of the layout work, but they are fiddly to style and not something we typically use anymore for layout.

If this approach gives you problems, you could restructure with HTML tags (see this [example of a comment using `article`](https://www.w3.org/TR/2013/CR-html5-20130806/sections.html#the-article-element)) and something like [flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) for layout.

You've clearly spent a lot of time structuring the tables, so I'd only do this if the tables get frisky and stop obeying.